### ISSUE TITLE
Lorem ipsum generator

### ISSUE THREAD

**Issue by isneezy** [akiraux/Akira#127]
First of all, I'd like thank you guys for bringing us a such great and promising tool into linux

It wold be great if we could generate lorem ipsum text inside the app, most of the time i find myself doing copy/paste dummy text from the web or typing some random text just to simulate real text.

**Comment by bilelmoussaoui** [comment]
That would be a pretty neat feature indeed. Having support for Lorem picsum would be great too.

**Comment by abienz** [comment]
Sketch has the feature for adding in Images from Unsplash, Random person names, and Randon City names, I feel like this sort of feature should aim to match what Sketch offers here.

**Comment by julientaq** [comment]
Unsplash has a neat API to get random pic. 

On the lorem ipsum, i would rather have some real content coming from real language. Maybe sanitizing some wikipedia article could be more useful than lorem ipsum (and would then get multi-lang content)

**Comment by isneezy** [comment]
While i totally agree with @julientaq idea, I'm afraid that I and many others live in contries where the internet connection n is not always reliable, it wold be wise if this functionality can always fallback to a traditional lorem ipsum if there is no connection, and also I've never used sketch but we can go deeper with @abienz suggestion and offer a lot more dummy kind of info like what "Faker" libraries does in many programming languages and may be we can write one for vala or c.

https://github.com/fzaninotto/Faker
https://github.com/DiUS/java-faker
https://github.com/stympy/faker

**Comment by Alecaddd** [comment]
@isneezy that's a really good point.
I definitely see this implementation as a plugin with many more features to be added.
We're gonna start working on a pluggable interface right after the first release, meanwhile, if any of you wants to start prototyping some ideas, we can use this issue to collect a list of initial features for the **Faker Copy** plugin...and maybe even find a cool name 😄

**Comment by rubaiyat** [comment]
Agree. Suggest as a **Lorem Tool** which fills a dragged area with text from standard sources (Head/Subhead/Body/List/bulleted-numbered list) flipped w keyboard shortcuts, and then snips the text at the end of the bounding box.

Effectively you paint in the text which immediately becomes styled placeholder text.

The same with a **Lorem picsum** Tool.

Another approach is to change the **Fill control** to also fill with text/image.

I do all this manually, with drag and drop text/image files. Having tools do this instead would be much faster and give the design a smoother interactive flow.

I can supply both my _Lorem Ipsum text files_ and _image placeholders_ if you like.

**Comment by Alecaddd** [comment]
> Another approach is to change the Fill control to also fill with text/image.

Not sure about text, but for sure we're planning to add an Image fill option to basically let the shape act as a clipping mask for the fill image.

**Comment by rubaiyat** [comment]
Maybe I need to mock up my ideas. :)

But if you are using Mac Apps and Sketch/Figma you may have some idea of what I am talking about. 

I'd like to see Akira actually become one of many reasons to switch to Linux.

Other than server management and some programming tools, Linux does not have many compelling reasons to switch.

Could I put my hand up to help document Akira? **Help** is very Mother Hubbard at the moment. :)

**Comment by Alecaddd** [comment]
> But if you are using Mac Apps and Sketch/Figma you may have some idea of what I am talking about.

I've been a designer for most of my life and I used those apps very extensively (and still do for work). Akira aims at being a perfect replacement for those.

> Could I put my hand up to help document Akira? 

Sure, any help is appreciated.
Being in Alpha state, a lot of things change very quickly, that's why I didn't spend too much time writing documentation as nothing is set in stone for now.
